### PR TITLE
Update php symfony version

### PR DESCRIPTION
**Pull request by ottaviano**
This PR updates PHP to 7.0 and Symfony from 2.4 to 2.8.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-install: build run
+install: build up composer check assetic dbinstall
 
-run: up composer check assetic
+run: up
```
Comment (HeahDude) [inline Makefile]: this alias seems useless now

File: Makefile [hunk 2]
```diff
@@ -20,3 +20,7 @@ down:
 
 assetic:
 	docker-compose run --rm php app/console assetic:dump
+
+dbinstall:
+	docker-compose run --rm php app/console doctrine:database:create
+	docker-compose run --rm php app/console doctrine:schema:create
```
Comment (HeahDude) [inline Makefile]: aren't you missing some flag?

Comment (ottaviano) [inline Makefile]: @HeahDude, do you mean `--env`?

File: Makefile (hunk 1 above)
Comment (ottaviano) [inline Makefile]: @HeahDude, removed.

**Pull request merged**